Dockerfile

```diff
@@ -26,6 +26,10 @@ RUN npm config set cafile "/app/DOIRootCA2.cer" \
 
 RUN node_modules/bower/bin/bower install --allow-root
 
+RUN nosetests --all-modules --exe
```

Comment:
Just reading about nosetests. Why are you using --exe? I haven't had to use this in the past.